File: tehnohem-api/DTO/OutgoingInvoiceDTO.cs
using tehnohem_api.Model.Enums;
using tehnohem_api.Model;

namespace tehnohem_api.DTO
{
    public class OutgoingInvoiceDTO
    {
        public string InvoiceID { get; set; }
        public string BuyerID { get; set; }
        public DateOnly Date { get; set; }
        public float TotalValue { get; set; }
        public float TotalValueOfPDV { get; set; }
        public float TotalValueWithoutPDV { get; set; }

        public ICollection<OutgoingInvoiceItemDTO> InvoiceItems { get; set; }


    }
}

File: tehnohem-ui/src/app/shared/model/invoices/outgoingInvoiceDTO.ts
import { ProductToSell } from "../productToSell";

export interface OutgoingInvoiceDTO {
    invoiceID: string ;
    isCashInvoice:boolean;
    buyerID: string ;
    date: string ;
    totalValue: number ;
    totalValueOfPDV: number ;
    totalValueWithoutPDV: number ;
    
    invoiceItems:ProductToSell[] ;
       
}